### PR TITLE
feat(logging): add interface to change logger

### LINKED ISSUE #3: Logging
Hi,

what would you think about to add an optional logging configuration? That will allow to pass in an own preferred logger which satisfy the standard go logging interface.

### PR DESCRIPTION
**Pull request by Nightapes**
Fixes #3

### CODE REVIEW

File: logger.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import (
+	"errors"
+	"log"
+	"os"
+)
+
+var errorLogger = Logger(log.New(os.Stderr, "[exasol] ", log.LstdFlags|log.Lshortfile))
+
+// Logger is used to log critical error messages.
```
Comment (AnastasiiaSergienko) [inline logger.go:11]: If this interface is only used to log errors, maybe it's more precise to name it ErrorLogger?

**Review by AnastasiiaSergienko — COMMENTED** [review]
Thanks for assigning me. I think reviews are a great way to get used to a new language. :) 
I just left a small suggestion

File: logger.go (hunk 1 above)
Comment (Nightapes) [reply logger.go:11]: I would keep it `Logger` if we want to use it for normal log messages in the future

Comment (AnastasiiaSergienko) [reply logger.go:11]: Okay. Then it's fine to merge

**Review by AnastasiiaSergienko — APPROVED** [review]

**Pull request merged**